Makefile

```diff
@@ -33,7 +33,7 @@ install_tomlfmt: install_rust
 ## User install
 
 install:
-	$(CARGO) install --force --path .
+	$(CARGO) install --force --path . $(BUILD_ARGS)
```

Comment:
I disagree with this. `install` is not the `build` command and neither are `clippy`, `test` and `doc`. If we want/need to customize those, I propose to add more variables.

`install` for example does a release build by default, so there is no need to pass this.

Reply:
This is not for the "release" part, it's for the feature flag!

Reply:
True, we should rename it to FEATURES then or something like that :)

Reply:
I'll incorporate that into my open PR!

Reply:
Actually, if we merge my open PR, you can just rebase this one and don't have to do anything else, I think :)